{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "ecd39fc5_aa7e4037",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 13252
      },
      "writtenOn": "2024-06-13T12:18:39Z",
      "side": 1,
      "message": "recheck multiple job failures likely due to pypi issue yesterday",
      "revId": "2e3fe61e88ad63682c94689ae994a9fa75954d19",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}